src/scipp/core/comparison.py

```diff
@@ -165,6 +166,12 @@ def identical(x: VariableLike,
         True if x and y have identical values, variances, dtypes, units,
         dims, shapes, coords, and masks. Else False.
     """
+    if isinstance(x, data_group.DataGroup):
+        if x.keys() != y.keys():
+            return False
+        results = data_group._data_group_binary(identical, x, y, equal_nan=equal_nan)
+        return all(results.values())
+
```

Comment:
Is there a better way to add support for DataGroup than to add an explicit `isinstance` to every function? Can we hook into `_call_cpp_func` and build an overload table of some sort?

Reply:
Good question, will look into it when we add support in more functions.

Reply:
Seems to work: #2959